test/token/ERC20/RecoverableToken.test.js

```diff
@@ -0,0 +1,39 @@
+const StandardTokenMock = artifacts.require('StandardTokenMock');
+const RecoverableTokenMock = artifacts.require('RecoverableTokenMock');
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('RecoverableToken', function ([creator, owner, recipient, anotherAccount]) {
```

Comment:
It's not really documented anywhere yet (I'm working on that :)), but we're introducing the convention of not using `accounts[0]`, since that's the default account, and only naming the rest. Could you change the function arguments to be `[_, creator, owner, recipient, anotherAccount]`, so that the first one is unused?

Reply:
In your signature the _ is the 'real' creator of the contract, when i'm test, maybe I make mistake can you explain ?